p4-16/spec/P4-16-spec.mdk

```diff
@@ -1304,8 +1304,8 @@ number of backslash characters (ASCII code 92). P4 does not make any
 validity checks on strings (i.e., it does not check that strings
 represent legal UTF-8 encodings).
 
-Since P4 does not provide any operations on strings,
-string literals are generally passed unchanged through the P4 compiler to
+Since P4 does not allow strings to exist at runtime, string literals
```

Comment:
I think this isn't really true anymore? Now we also concatenate them e.g.? And I'm guessing we plan to turn that into a single string literal before passing it along to backends (though maybe not?). 

I think the point of this paragraph is to say that the P4 compiler won't interpret string literals. What does that mean in the new world with concatenation?

Reply:
Probably terminating quotes are no longer left as-is?